sources/api/migration/migrator/src/error.rs

```diff
@@ -71,9 +71,7 @@ pub(crate) enum Error {
     ListMigrations { dir: PathBuf, source: io::Error },
 
     #[snafu(display("Error loading manifest: {}", source))]
-    LoadManifest {
-        source: update_metadata::error::Error,
-    },
+    LoadManifest { source: tough::error::Error },
```

Comment:
supernit: If we're changing this anyway, maybe it could be ManifestLoad to fit the pattern and sit next to the other Manifest* errors?  :)